lib/node_modules/@stdlib/blas/ext/base/zfill/README.md

```diff
@@ -173,9 +173,11 @@ var im = imag( y );
 
 The function has the following additional parameters:
 
--   **offset**: starting index.
+-   **offsetX**: starting index.
 
-While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying `buffer`, the offset parameter supports indexing semantics based on a starting index. For example, to access only the last two elements of the strided array
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter supports indexing semantics based on a starting index. For example, to access only the last two elements of the strided array
+
+<!-- eslint-disable stdlib/doctest -->
```

Comment:
@kgryte Need some insight into this. The `doctest` for this JS example fails while all the test cases pass. I tried removing all the changes but same result.

Reply:
You need to provide me the error message you are seeing.

Reply:
```git commit -m "feat: add C ndarray API and refactor"

Linting file: lib/node_modules/@stdlib/blas/ext/base/cfill/README.md

Linting file: lib/node_modules/@stdlib/blas/ext/base/zfill/README.md
lib/node_modules/@stdlib/blas/ext/base/zfill/README.md
  180:1-220:4  error  19:1   error   Displayed return value is `1.0`, but expected `10` instead   stdlib/doctest  eslint  remark-lint
  180:1-220:4  error  22:1   error   Displayed return value is `2.0`, but expected `10` instead   stdlib/doctest  eslint  remark-lint
  180:1-220:4  error  28:1   error   Displayed return value is `10.0`, but expected `5` instead   stdlib/doctest  eslint  remark-lint
  180:1-220:4  error  31:1   error   Displayed return value is `10.0`, but expected `6` instead   stdlib/doctest  eslint  remark-lint

✖ 4 errors
make: *** [/home/odin/Documents/stdlib/tools/make/lib/lint/markdown/Makefile:72: lint-markdown-files] Error 1

Markdown lint errors.```

Reply:
Is the native add-on compiled when you are attempting to commit? If so, have you tried removing the native add-on and seeing if the error message persists? In short, you want to pin down whether this is coming from the JS or native implementation or whether it applies to both.

Reply:
Looks like this is due to a bug in the C `ndarray` implementation, as commented on below.